frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1087,17 +1086,15 @@ export default ({
     }, 250),
     // Handlers for file-upload via drag & drop action
     dragStartHandler (e) {
+      e.preventDefault()
       // handler function for 'dragstart', 'dragover' events
-      const items = Array.from(e?.dataTransfer?.items) || []
-
-      if (items.some(entry => entry.kind === 'file')) { // check if the detected content is something attachable to the chat.
-        if (!this.dndState.isActive) {
-          this.dndState.isActive = true
-        }
```

Comment:
(If my memory is correct) This check was added in an attempt to prevent a folder from being attached to the chat. But actually `filtering out directories` logic ended up  being moved to `SendArea.vue` ([here](https://github.com/okTurtles/group-income/blob/cb73265fd9712e1f89da5cab68e0234d65412427/frontend/views/containers/chatroom/SendArea.vue#L694-L703)) later on, so this is not needed here anymore.

The cause of the issue was that the `e?.dataTransfer?.items` of `dragover` event here gets populated in chrome / FF but does not in Safari, btw.